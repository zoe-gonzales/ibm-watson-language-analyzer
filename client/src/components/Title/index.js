import React from 'react';

const Title = props => {
    return <h3>{props.children}</h3>
}

export default Title;