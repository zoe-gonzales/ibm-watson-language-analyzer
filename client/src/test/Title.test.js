import React from 'react';
import Title from '../components/Title';
import renderer from 'react-test-renderer';

it('renders Title component correctly', () => {
  const component = renderer
  .create(<Title>Text Analyzer</Title>)
  .toJSON();
  expect(component).toMatchSnapshot();
});