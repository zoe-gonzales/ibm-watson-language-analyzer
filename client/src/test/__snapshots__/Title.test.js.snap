// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Title component correctly 1`] = `
<h3>
  Text Analyzer
</h3>
`;
